src/main/java/org/gradle/profiler/report/JsonResultWriter.java

```diff
@@ -24,23 +32,29 @@ public JsonResultWriter(boolean pretty) {
         this.pretty = pretty;
     }
 
-    public void write(List<? extends BuildScenarioResult<?>> scenarios, Writer writer) {
+    public void write(List<? extends BuildScenarioResult<?>> scenarios, Temporal reportDate, Writer writer) {
```

Comment:
The report date probably should be a `ZonedDateTime`. I think that is what we want to use here.

Reply:
Why? We are using an UTC `Instant` in order to remove any environment-dependence in the output to make testing simpler. If you think this should be somewhat different, please propose the commits to make it so. I'm also fine with reverting to `Date.

Reply:
I think the report should show the User's local date and time, I can do some changes to that regard. You can go ahead and merge this PR as is.

Reply:
I think the JSON should be a source of truth regardless of where the benchmark was executed, and as such it should have UTC times. Then the HTML should show it in the local time zone. This is what happens currently.

Reply:
Sounds good!